<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FutureGUIClass</class>
 <widget class="QMainWindow" name="FutureGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FutureGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>800</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="FutureGUI.qrc">:/FutureGUI/jugenstilBG.png</pixmap>
    </property>
   </widget>
   <widget class="MaskedButton" name="specialBtnR">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>55</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/FutureGUI/jugenstilArrowRight.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>38</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="MaskedButton" name="stalkBtnR">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>470</y>
      <width>55</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/FutureGUI/jugenstilArrowRight.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>38</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="MaskedButton" name="rootBtnR">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>650</y>
      <width>55</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/FutureGUI/jugenstilArrowRight.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>38</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="MaskedButton" name="leafBtnR">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>55</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/FutureGUI/jugenstilArrowRight.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>38</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="MaskedButton" name="leafBtnL">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>55</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/FutureGUI/jugenstilArrowLeft.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>38</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="MaskedButton" name="specialBtnL">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>55</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/FutureGUI/jugenstilArrowLeft.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>38</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="MaskedButton" name="rootBtnL">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>650</y>
      <width>55</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/FutureGUI/jugenstilArrowLeft.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>38</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="MaskedButton" name="stalkBtnL">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>55</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/FutureGUI/jugenstilArrowLeft.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>38</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="MaskedButton" name="saveBtn">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>20</y>
      <width>149</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/FutureGUI/jugenstilSaveBtn.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>149</width>
      <height>90</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MaskedButton</class>
   <extends>QPushButton</extends>
   <header>MaskedButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="FutureGUI.qrc"/>
 </resources>
 <connections/>
</ui>
